src/inject.js

```diff
@@ -203,9 +203,13 @@ export const reset = () => {
     styleTag = null;
 };
 
+export const resetInjectedStyle = (key /* : string */) => {
+    delete alreadyInjected[key];
```

Comment:
Would it be sufficient to do this? 

```js
alreadyInjected[key] = undefined;
```

or do we have code that relies on it being fully deleted?

Reply:
We do an `Object.keys(alreadyInjected)`, so deleting seems like the best approach.